source/User_Guide/Marketing_Campaigns/Managing_Contacts/managing_contact_list.md

```diff
@@ -45,8 +44,8 @@ When you are viewing a contact, click any of the Edit buttons to edit the contac
 1. Enter or select the list you would like to use.
 1. Click **Add**.
 
-{% anchor h3 %}
-Managing Your List
+{% anchor h2 %}
+Managing your list
 {% endanchor %}
 
```

Comment:
I would add a mini table of contents here, since this is a long and wordie list that's difficult to scan